redaxo/src/core/lib/extension.php

```diff
@@ -27,7 +27,7 @@ abstract class rex_extension
      *
      * @template T
      * @param rex_extension_point<T> $extensionPoint Extension point
-     * @return (T is null ? mixed : T) Subject, maybe adjusted by the extensions
+     * @return T Subject, maybe adjusted by the extensions
```

Comment:
Mir ist jetzt aufgefallen, dass der Fix an anderen Stellen Probleme schafft. Z.B. hier:
https://github.com/redaxo/redaxo/blob/84d15882152edc896422d74ebe208e65ee62b5bb/redaxo/src/core/lib/util/editor.php#L83

`$editorUrl` hat an der Stelle den Typ `string|null`. Darauf matcht aber auch unser `T is null`.
Eigentlich wollten wir damit ja nur Fälle abfangen, wo `$subject` nur `null` ist, und nicht Fälle, wo `null` Teil eines Union-Types ist.

Ich würde den Fix aus dem anderen PR hier daher rückgängig machen. Denn sehe für die neu problematischen Stellen keinen guten Fix. Und die Stellen, die mit dem Fix gelöst wurden, die kann man hingegen einigermaßen sinnvoll auch anders lösen.